src/utils/db.ts

```diff
@@ -1,8 +1,23 @@
 import pgPromise from "pg-promise"
 
-const dbUrl: string = process.env.DATABASE_URL ? process.env.DATABASE_URL + "?ssl=true" : ""
+// Disable SSL if in development mode
+function getDbString(): string {
+    const baseURL = process.env.DATABASE_URL ? process.env.DATABASE_URL: ""
+    if (process.env.NODE_ENV === "development") {
+        return baseURL
+    } else {
+        return baseURL + "?ssl=true"
+    }
+}
+const dbUrl: string = getDbString()
+
+// Skip SSL verification if the environment is dev
+const ssl = (process.env.NODE_ENV === "development") ? {rejectUnauthorized: false} : {rejectUnauthorized: true}

```

Comment:
I think this should be
```typescript
const ssl = {
    rejectUnauthorized: process.env.NODE_ENV !== "development"
}
```

or something like
```typescript
const rejectUnauthorized = process.env.NODE_ENV !== "development"
const ssl = {
    rejectUnauthorized
}
```

Reply:
I'll adjust this